### PR TITLE
Add test for validating dynamic aggregates title.

### PR DESCRIPTION
**Pull request by Piotrk39**
## Describe your changes

Adds E2E test for validating aggregates title to be dynamic. 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review

### CODE REVIEW

File: tests/e2e/Pages/AggregatesComponet.ts
Comment (gulfaraz) [inline tests/e2e/Pages/AggregatesComponet.ts:1]: The new spelling is missing a n. Component.

File: tests/e2e/Pages/MapComponent.ts [hunk 1]
```diff
@@ -115,6 +120,36 @@ class MapComponent extends DashboardPage {
...
+    // Declare component
+    const aggregates = new AggregatesComponent(this.page);
+
+    // Wait for the page to load
+    await this.page.waitForLoadState('networkidle');
+    await this.page.waitForLoadState('domcontentloaded');
+    await this.page.waitForSelector('.leaflet-interactive');
+
+    // Assert the that Aggregates title is visible and does not contain the text 'National View'
+    // 20 is the more or less number of admin boundaries that can be hovered over with Playwright to succesfully assert the title
```
Comment (gulfaraz) [inline tests/e2e/Pages/MapComponent.ts:144]: I don't understand this magic number 20. Is it not sufficient to test one admin boundary and assume the others work the same way?

I feel I'm missing some testing-specific science here so correct me if I'm wrong.

File: tests/e2e/helpers/utility.helper.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import * as request from 'supertest';
 import TestAgent from 'supertest/lib/agent';
 
-import { FloodsScenario } from '../../../services/API-service/src/scripts/enum/mock-scenario.enum';
```
Comment (gulfaraz) [inline tests/e2e/helpers/utility.helper.ts:4]: Nice!

**Review by gulfaraz — APPROVED** [review]

File: tests/e2e/Pages/MapComponent.ts (hunk 1 above)
Comment (Piotrk39) [reply tests/e2e/Pages/MapComponent.ts:144]: I thought about it and it does not make sense to put any number there. Initially I wanted it to loop over all the components but I was facing a huge timeouts. I'll change it to validate to one. Also not all disaster types will have 20 regions displayed.

File: tests/e2e/Pages/AggregatesComponet.ts
Comment (Piotrk39) [reply tests/e2e/Pages/AggregatesComponet.ts:1]: Three times a charm. Fixed ;)

File: tests/e2e/Pages/MapComponent.ts (hunk 1 above)
Comment (gulfaraz) [reply tests/e2e/Pages/MapComponent.ts:144]: Thank you for clarifying @Piotrk39

**Pull request merged**